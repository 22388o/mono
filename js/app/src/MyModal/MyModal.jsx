import React from 'react'
import { Box, Modal } from '@mui/material'

export const MyModal = ({ classme, children, open, handleClose = () => {} }) => {
  return (
    <Modal
      open={open}
      onClose={handleClose}
      aria-labelledby='parent-modal-title'
      aria-describedby='parent-modal-description'
      sx={{ width: 'fit-content', display: 'flex', margin: 'auto' }}
    >
      <Box className={`${classme} modal-container`}>
        {children}
      </Box>
    </Modal>
  )
}
