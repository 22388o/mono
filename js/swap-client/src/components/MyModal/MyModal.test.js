import { create, act } from 'react-test-renderer'
import { MyModal } from './MyModal'

describe('MyModal test', () => {
  let tree
  it('should work', () => {
    act(() => {
      tree = create(
        <MyModal
          classme={null}
          children={null}
          open={false}
        />
      )
    })

    expect(tree).toMatchSnapshot()
  })
  afterAll(() => jest.resetModules())
})
