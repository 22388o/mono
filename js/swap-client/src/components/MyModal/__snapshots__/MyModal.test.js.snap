// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyModal test should work 1`] = `null`;
